src/sentry/integrations/jira/utils/api.py

```diff
@@ -75,26 +76,40 @@ def handle_assignee_change(
     sync_group_assignee_inbound(integration, email, issue_key, assign=True)
 
 
-def handle_status_change(integration, data):
-    issue_key = data["issue"]["key"]
-    status_changed = any(item for item in data["changelog"]["items"] if item["field"] == "status")
-    log_context = {"issue_key": issue_key, "integration_id": integration.id}
-
-    if not status_changed:
-        logger.info("jira.handle_status_change.unchanged", extra=log_context)
-        return
-
-    try:
-        changelog = next(item for item in data["changelog"]["items"] if item["field"] == "status")
-    except StopIteration:
-        logger.info("jira.missing-changelog-status", extra=log_context)
-        return
-
-    result = integration_service.organization_contexts(integration_id=integration.id)
-    for oi in result.organization_integrations:
-        install = integration.get_installation(organization_id=oi.organization_id)
-        if isinstance(install, IssueSyncIntegration):
-            install.sync_status_inbound(issue_key, {"changelog": changelog, "issue": data["issue"]})
+# TODO(Gabe): Consolidate this with VSTS's implementation, create DTO for status
+# changes.
+def handle_status_change(integration: RpcIntegration, data: Mapping[str, Any]) -> None:
+    with ProjectManagementEvent(
+        action_type=ProjectManagementActionType.INBOUND_STATUS_SYNC, integration=integration
+    ).capture() as lifecycle:
+        issue_key = data["issue"]["key"]
+        status_changed = any(
+            item for item in data["changelog"]["items"] if item["field"] == "status"
+        )
+        log_context = {"issue_key": issue_key, "integration_id": integration.id}
+
+        if not status_changed:
+            logger.info("jira.handle_status_change.unchanged", extra=log_context)
+            return
+
+        try:
+            changelog = next(
+                item for item in data["changelog"]["items"] if item["field"] == "status"
+            )
+        except StopIteration:
+            lifecycle.record_halt("missing-changelog-status", extra=log_context)
```

Comment:
extreme nit - do we want to put all the reasons in an enum or something?

Reply:
Fair! I'll do that now.